integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisRestCatalogIntegrationTest.java

```diff
@@ -1012,4 +1012,26 @@ public void testRenameViewStatus() {
       assertThat(response).returns(Response.Status.NO_CONTENT.getStatusCode(), Response::getStatus);
     }
   }
+
+  @Test
+  public void testLoadCredentials() {
+    String tableName = "tbl1";
+    Namespace namespace = Namespace.of("ns1");
+    TableIdentifier identifier = TableIdentifier.of(namespace, tableName);
+
+    if (requiresNamespaceCreate()) {
```

Comment:
nit: Isn't this always `true` for Polaris?

Reply:
I think so, and it is on my machine, but I wondered if that was due to my environment not correctly garbage-collecting a namespace created elsewhere. I copy/paste'd this from another test in this suite. I can try to remove it and see if we pass CI